item-ids.md

```diff
@@ -0,0 +1,130 @@
+# Item IDs
+
+Brief and likely incomplete list of the inventory/pickup items.
+
+This revision does not account for the inventory items responsible for options, loading, saving, previewing IQ points, etc.
+
+| ID  | What it is                                                           |
+|:---:|:--------------------------------------------------------------------:|
+| 0   | (nothing)                                                            |
+| 1   | Boxing gloves/Fists                                                  |
+| 2   | Whip                                                                 |
+| 3   | Revolver                                                             |
+| 4   | Tokarev                                                              |
+| 5   | Mauser                                                               |
+| 6   | Simonov SKS/Combat Rifle                                             |
+| 7   | Machete                                                              |
+| 8   | PPSH-41/Submachine Gun                                               |
+| 9   | TOZ-34/Pump Shotgun                                                  |
+| 10  | Bazooka                                                              |
+| 11  | Grenade                                                              |
+| 12  | Satchel Charge                                                       |
+| 13  | Zippo lighter                                                        |
+| 14  | IMP1/Urgon's Part/The Rumbler                                        |
+| 15  | IMP2/Taklit's Part/The Invisibility Part                             |
+| 16  | IMP3/Azerim's Part/The Levitator                                     |
+| 17  | IMP4/Nub's Part/The Power Part                                       |
+| 18  | IMP5/Tool from Beyond/The Reality Dialator                           |
+| 19  | (nothing)                                                            |
+| 20  | (nothing)                                                            |
+| 21  | (nothing)                                                            |
+| 22  | (nothing)                                                            |
+| 23  | (nothing)                                                            |
+| 24  | (nothing)                                                            |
```

Comment:
Just fyi the 19 -24 are items AI actors have (commies). Tho I guess can't be used by player so it's ok not to be on the list.